BParkingNano/plugins/ElectronMerger.cc

```diff
@@ -172,15 +242,36 @@ void ElectronMerger::produce(edm::StreamID, edm::Event &evt, edm::EventSetup con
   /// add and clean low pT e
   for(auto ele : *lowpt) {
     iele++;
+
+   // regression stuff
+   // regression stuff
+   float pre_ecal = ele.superCluster()->rawEnergy();
+   float pre_ecaltrk = ele.gsfTrack()->pMode();
+   regression_->modifyObject(ele);
+   float post_ecal = ele.correctedEcalEnergy();
+   float post_ecaltrk = ele.energy();
+
+   if(debug)std::cout <<"LP ele pre SC/TK --> post SC/TK"<< pre_ecal<<"/"<<pre_ecaltrk 
+		      << " -> " << post_ecal<<"/"<<post_ecaltrk << std::endl;                                                                                        
+   // end regression stuff                                                                                                                                   
+
+
+
     //take modes
-   if(use_gsf_mode_for_p4_) {
+   if (use_regression_for_p4_) {
```

Comment:
Perhaps it would be useful to add a fail-safe. in case somebody accidentaly add them both